core/offchain-worker-executor/src/executor.rs

```diff
@@ -218,24 +230,83 @@ mod tests {
 		TestStateHandler,
 		TestValidatorAccess,
 		TestExtrinsicsFactory,
+		TestStfInterface,
 	>;
 
+	static EVENT_COUNT_KEY: &[u8] = b"event_count";
```

Comment:
Just a small detail: I think `const` would be a better choice here than `static`.